libs/auth-api-lib/src/lib/services/clients.service.ts

```diff
@@ -81,6 +86,9 @@ export class ClientsService {
       limit: count,
       offset: offset,
       distinct: true,
+      where: {
+        archived: null,
+      },
     })
   }
 
```

Comment:
Should there be a function to view archived Clients or Api Resources?

Reply:
Removed as requested